bifacialvf/sun.py

```diff
@@ -378,7 +378,7 @@ def solarPos( year, month, day, hour, minute, lat, lng, tz ):
 			jday = julian(year,month,day);		# Get julian day of year
 			zulu = hour + minute/60.0 - tz;		# Convert local time to zulu time
 			delta = year - 1949;
-			leap = delta/4;
+			leap = (int)(delta/4);
```

Comment:
This looks like C code.
- this works fine, but a more Pythonic way is just `int(delta/4)`.
- Also I think if you do `delta // 4` it will still be a float, but the value will look like an `int`

```python
>>> 23.45/4
5.8625
>>> (int)(23.45/4)
5
>>> int(23.45/4)
5
>>> 23.45//4
5.0
```

Reply:
I'll take Mark's way.